app/queries/consumer_apps/find_or_create_all_query.rb

```diff
@@ -5,8 +5,15 @@ def self.call
       existing_platforms = ConsumerApp.where(app_bundle: forem_bundle).pluck(:platform)
       (ConsumerApp::FOREM_APP_PLATFORMS - existing_platforms).each do |platform|
         # Re-create the supported Forem apps if they're missing
-        ConsumerApp.create(app_bundle: forem_bundle, platform: platform, active: true,
-                           team_id: ConsumerApp::FOREM_TEAM_ID)
+        ConsumerApp.create!(app_bundle: forem_bundle, platform: platform, active: true,
+                            team_id: ConsumerApp::FOREM_TEAM_ID)
+      rescue StandardError => e
+        error_tags = [
+          "error:#{e.message}",
+          "app_bundle:#{forem_bundle}",
+          "platform:#{platform}",
+        ]
+        ForemStatsClient.increment("consumer_apps.create", 1, tags: error_tags)
```

Comment:
Oooh, I like this! 🤩

Reply:
There was a typo in there, but almost ready now! 😄